src/main/java/com/gotocompany/depot/metrics/MaxComputeMetrics.java

```diff
@@ -37,4 +37,7 @@ public String getMaxComputeFlushRecordMetric() {
         return String.format("%s%s%s%s", getApplicationPrefix(), SINK_PREFIX, MAXCOMPUTE_SINK_PREFIX, "flush_record_count");
     }
 
+    public String getMaxComputeConversionLatencyMetric() {
```

Comment:
Can we also add metrics to track the number of upload sessions created by firehose?